tools/helper_default_based_schedule

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/perl
+
+# Tool to merge YAML_SCHEDULE_DEFAULT  with YAML_SCHEDULE_FLOWS
+# and YAML_SCH§EDULE.
+
+# So far all pointers should be provided as environment
+# variables
+
+use strict;
+use warnings;
+
+use File::Basename;
+
+use Env;
```

Comment:
could you please check the use all the imports, I think some of them are not needed.